packages/rocketchat-api/server/v1/users.js

```diff
@@ -208,7 +208,10 @@ RocketChat.API.v1.addRoute('users.update', { authRequired: true }, {
 
 			const userData = _.extend({ _id: this.bodyParams.userId }, this.bodyParams.data);
 
+			///////////// Please review this line. It will fail if we just want to enable user using REST API
+			////////////  {"userId": "ID", "data": { "active": true }}
 			RocketChat.saveUser(this.userId, userData);
```

Comment:
Then you can go ahead and remove your comments as Pull Request #5402 fixed that issue.